diff --git a/Battery Toolkit.xcodeproj/project.pbxproj b/Battery Toolkit.xcodeproj/project.pbxproj
index 7b92a1c..0a7300c 100644
--- a/Battery Toolkit.xcodeproj/project.pbxproj	
+++ b/Battery Toolkit.xcodeproj/project.pbxproj	
@@ -876,7 +876,7 @@
 				ASSETCATALOG_COMPILER_GENERATE_SWIFT_ASSET_SYMBOL_EXTENSIONS = YES;
 				BT_APP_ID = me.mhaeuser.BatteryToolkit;
 				BT_AUTOSTART_ID = me.mhaeuser.BatteryToolkitAutostart;
-				BT_CODESIGN_CN = "Apple Development: Marvin Häuser (87DYA6FH9K)";
+				BT_CODESIGN_CN = "Apple Development: bryanlai@foxmail.com (VY3W9R894Q)";
 				BT_DAEMON_CONN = EMH49F8A2Y.me.mhaeuser.batterytoolkitd;
 				BT_DAEMON_ID = me.mhaeuser.batterytoolkitd;
 				BT_SERVICE_ID = me.mhaeuser.BatteryToolkitService;
@@ -940,7 +940,7 @@
 				CURRENT_PROJECT_VERSION = 1.7;
 				DEAD_CODE_STRIPPING = YES;
 				DEBUG_INFORMATION_FORMAT = dwarf;
-				DEVELOPMENT_TEAM = EMH49F8A2Y;
+				DEVELOPMENT_TEAM = AWMJ8H4G7B;
 				ENABLE_HARDENED_RUNTIME = YES;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
@@ -1013,7 +1013,7 @@
 				ASSETCATALOG_COMPILER_GENERATE_SWIFT_ASSET_SYMBOL_EXTENSIONS = YES;
 				BT_APP_ID = me.mhaeuser.BatteryToolkit;
 				BT_AUTOSTART_ID = me.mhaeuser.BatteryToolkitAutostart;
-				BT_CODESIGN_CN = "Apple Development: Marvin Häuser (87DYA6FH9K)";
+				BT_CODESIGN_CN = "Apple Development: bryanlai@foxmail.com (VY3W9R894Q)";
 				BT_DAEMON_CONN = EMH49F8A2Y.me.mhaeuser.batterytoolkitd;
 				BT_DAEMON_ID = me.mhaeuser.batterytoolkitd;
 				BT_SERVICE_ID = me.mhaeuser.BatteryToolkitService;
@@ -1077,7 +1077,7 @@
 				CURRENT_PROJECT_VERSION = 1.7;
 				DEAD_CODE_STRIPPING = YES;
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
-				DEVELOPMENT_TEAM = EMH49F8A2Y;
+				DEVELOPMENT_TEAM = AWMJ8H4G7B;
 				ENABLE_HARDENED_RUNTIME = YES;
 				ENABLE_NS_ASSERTIONS = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
